### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Mcdavid95**

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: 17EGfAB9q5pSxYN9XcAMgbd0zGo3EzXoq
```
Comment (andela-gike) [inline .coveralls.yml]: It is not advisable to push your repo token to github as one of your files

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach to Port",
+      "address": "localhost",
+      "port": 5858
+    }
+  ]
+}
```
Comment (andela-gike) [inline .vscode/launch.json]: Your are not to push your .vscode file to github

File: dist/inverted-index.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        searchWords[this.fileName] = results;
+      }
+      return searchWords;
+    }
+  }]);
+
+  return InvertedIndex;
+}();
+
+exports.default = InvertedIndex;
```
Comment (andela-gike) [inline dist/inverted-index.js]: You are not to push your dist folder to github

File: gulpfile.babel.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import gulp from 'gulp';
+import dotenv from 'dotenv';
+// import coverage from 'gulp-coverage';
```
Comment (andela-gike) [inline gulpfile.babel.js]: Don't add commented code when you are pushing to github

File: tests/inverted-index-tests.spec.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
+// import jasmine from 'jasmine-node';
+import InvertedIndex from '../src/inverted-index';
+import books from '../fixtures/books.json';
+import invalid from '../fixtures/invalidBook.json';
+import fortress from '../fixtures/book1.json';
+import server from '../routes/app';
+// import badBooks from '../fixtures/badBooks.json';
```
Comment (andela-gike) [inline tests/inverted-index-tests.spec.js]: Don't push commented code to github

**Review by EmaEvidence — COMMENTED** [review]
Good Job, your code is readable but try to increase the number of your tests.

**Comment by andela-gike** [comment]
You also have merge conflict resolve it

File: src/inverted-index.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    const words = check.split(',');
+    const searchWords = {};
+    const results = {};
+    if (words.length > 1) {
+      words.forEach((word) => {
+        if (word in index) {
+          results[word] = index[word];
+          searchWords[this.fileName] = results;
+        } else {
+          results[word] = ['Term not found'];
```
Comment (codingpop) [inline src/inverted-index.js:97]: In my opinion, I think you should be returning an empty array here

File: .coverrun [hunk 7]
```diff
@@ -0,0 +1 @@
+{ "run" : ".cover_7168115296224935" }
```
Comment (codingpop) [inline .coverrun:1]: You shouldn't be pushing this

**Comment by Mcdavid95** [comment]
thanks i will

**Pull request merged**